src/logic/field.h

```diff
@@ -25,8 +25,9 @@
 
 #include "base/wexception.h"
 #include "graphic/playercolor.h"
+#include "graphic/road_segments.h"
+#include "logic/map_objects/walkingdir.h"
```

Comment:
WalkingDir is not used in this header file.

Reply:
It´s used in line 182ff

Reply:
Thanks, I didn't didn't find it because it's an enum, not an enum class.